release-notes/opensearch-release-notes-2.15.0.md

```diff
@@ -0,0 +1,740 @@
+* Enhanced UI for rendering multiple documents in finding details flyout. ([#1014](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/1014))
+* [MDS][Part 2] Added server-side glue code to use data source id when getting opensearch client for making API calls to cluster. ([#1008](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/1008))
+* [MDS][Part 3] Wired all UI components to the data source menu. ([#1029](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/1029))
+
+
+### Opensearch k-NN
+
+
+* Use the Lucene Distance Calculation Function in Script Scoring for doing exact search [#1699](https://github.com/opensearch-project/k-NN/pull/1699)
+
+
+### OpenSearch SQL
+
+
+* Support Percentile in PPL ([#2710](https://github.com/opensearch-project/sql/pull/2710))
+
+
+## ENHANCEMENTS
+
+
+### Opensearch Alerting
+
+
+* Add `start_time` and `end_time` to GetAlertsRequest. ([#1551](https://github.com/opensearch-project/alerting/pull/1551))
+* Add Alerting Comments experimental feature ([#1561](https://github.com/opensearch-project/alerting/pull/1561))
+
+
+### Opensearch Anomaly Detection
+
+
+* Refinement of Forecasting and AD Precision/Recall Improvements ([#1210](https://github.com/opensearch-project/anomaly-detection/pull/1210))
+* Make jvm heap usage a dynamic setting ([#1212](https://github.com/opensearch-project/anomaly-detection/pull/1212))
+* Add custom result index lifecycle management condition fields to config ([#1215](https://github.com/opensearch-project/anomaly-detection/pull/1215))
+* Make Custom Result Index Name an Alias ([#1225](https://github.com/opensearch-project/anomaly-detection/pull/1225))
+* Add custom result index lifecycle management ([#1232](https://github.com/opensearch-project/anomaly-detection/pull/1232))
+* Merge Single-Stream and HC Detector Profiling Workflows ([#1237](https://github.com/opensearch-project/anomaly-detection/pull/1237))
+
+
+### Opensearch Anomaly Detection Dashboards
+
+
+* Update Frontend for Custom Result Index Query and Fix Issues ([#772](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/772))
+
+
+### OpenSearch Build
+
+
+* Enhance the stability of integration test runs, achieving passing test results on plugins of the distribution builds without requiring manual sign-offs.
+  * Add condition for OSD integ tests on deb and rpm ([#4770](https://github.com/opensearch-project/opensearch-build/pull/4770))
+  * Make endpoint_string one line for ccr test ([#4794](https://github.com/opensearch-project/opensearch-build/pull/4794))
+  * Add ci-groups changes for OSD tests ([#4796](https://github.com/opensearch-project/opensearch-build/pull/4796))
+  * Add cleanup for the data dir after integtests ([#4798](https://github.com/opensearch-project/opensearch-build/pull/4798))
```

Comment:
```suggestion
* Enhance the stability of integration test runs, achieving passing test results on plugins of the distribution builds without requiring manual sign-offs.
* Add condition for OSD integ tests on deb and rpm ([#4770](https://github.com/opensearch-project/opensearch-build/pull/4770))
* Make endpoint_string one line for ccr test ([#4794](https://github.com/opensearch-project/opensearch-build/pull/4794))
* Add ci-groups changes for OSD tests ([#4796](https://github.com/opensearch-project/opensearch-build/pull/4796))
* Add cleanup for the data dir after integtests ([#4798](https://github.com/opensearch-project/opensearch-build/pull/4798))
```

Reply:
Hi @gaiksaya all those two PRs are under the same category of `enhancing integtest`.